studygroups/models.py

```diff
@@ -599,6 +622,50 @@ def send_facilitator_survey(study_group):
         notification.send()
 
 
+def send_last_week_group_activity(study_group):
+    """ send to facilitator when last meeting is in 2 days """
+    now = timezone.now()
+    today = now.replace(hour=0, minute=0, second=0, microsecond=0)
+    two_days_from_now = today + datetime.timedelta(days=2);
+    last_meeting = study_group.meeting_set.active()\
+            .order_by('-meeting_date', '-meeting_time').first()
+
+    if last_meeting and last_meeting.meeting_datetime() > today and last_meeting.meeting_datetime() < two_days_from_now:
+
+        two_weeks_from_now = today + datetime.timedelta(weeks=2)
+        next_study_group = StudyGroup.objects.filter(start_date__gte=two_weeks_from_now).order_by('start_date').first()
+
+        timezone.deactivate()
+
+        context = {
+            'next_study_group': next_study_group,
+            'facilitator_name': study_group.facilitator.first_name
+        }
+
+        if next_study_group:
+            next_study_group_start_delta = next_study_group.start_date - today.date()
+            weeks_until_start = int(next_study_group_start_delta.days/7)
```

Comment:
@s-kennedy in Python 3+ you can do explicit integer division using `//` ie `next_study_group_start_delta.days//7`